docs/reference/query-rules/apis/put-query-rule.asciidoc

```diff
@@ -102,9 +105,9 @@ If multiple matching rules pin more than 100 documents, only the first 100 docum
 [[put-query-rule-example]]
 ==== {api-examples-title}
 
-The following example creates a new query rule with the ID `my-rule1` in a query ruleset called `my-ruleset`.
+The following example creates a new query rule in a query ruleset called `my-ruleset`.
```

Comment:
I'd still mention something regarding `my-rule1` and where that comes from (I assume it's an arbitrary ID for the name of the rule within the `my-ruleset` ruleset?)